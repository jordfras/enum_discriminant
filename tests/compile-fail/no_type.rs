use enum_discriminant::discriminant;

#[discriminant()]
enum EnumWithNoType {
    Unit = 17,
}

fn main() {}
