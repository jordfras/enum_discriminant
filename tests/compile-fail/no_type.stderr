error: Valid enum representation type expected as argument to the discriminant macro, e.g., #[discriminant(u8)]
 --> tests/compile-fail/no_type.rs:3:1
  |
3 | #[discriminant()]
  | ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `discriminant` (in Nightly builds, run with -Z macro-backtrace for more info)
